compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/CompilerPluginRunner.java

```diff
@@ -97,6 +87,15 @@ private CompilerPluginRunner(CompilerContext context) {
         this.processorMap = new HashMap<>();
     }
 
+    public static CompilerPluginRunner getInstance(CompilerContext context) {
```

Comment:
Why did you change the order of this function? We've been putting the getInstance function before the constructor in all the other classes?